modules/architecture-overview.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='architecture-overview-{context}']
+= Architecture overview
+
+Like {product-title} v3, {product-title} v4 is a layered system designed to
+expose underlying container images and Kubernetes concepts as accurately as
+possible, with a focus on easy composition of applications by a developer.
+For example, after your cluster is running, install Ruby, push code, and add
+MySQL to the app.
+
+A major difference between {product-title} v3 and v4 is that v4 uses Operators
+as both the fundamental unit of the product and an option for easily deploying
+and managing utilities that your apps use.
+
+{product-title} has an Operator-based architecture of smaller, decoupled units
+that work together. It runs on top of a Kubernetes cluster, with data about the
+objects stored in etcd, a reliable clustered key-value store. Those services are
+broken down by function:
+
+* Operators, which run the core {product-title} services.
+* REST APIs, which expose each of the core objects:
+** Containers and images, which are the building blocks for deploying your
+applications.
+** Pods and services, which containers use to communicate with each other and
+proxy connections.
+** Projects and users, which provide the space and means for communities to
+organize and manage their content together.
+** Builds and image streams] allow you to
+build working images and react to new images.
+** Deployments, which expand support for the software development and deployment
+lifecycle.
+** Routes, which announce your service to the world.
+** Templates, which allow you to simultaneously create many objects that are
```

Comment:
It’s time to deemphasoze templates - we need to describe our catalog of supporting application infrastructure as having three parts - operators (expose apis that automate full component lifecycle like databases), service bindings (consume services running elsewhere), and templates (simple instant examples).